ui/src/main/res/values/strings.xml

```diff
@@ -257,4 +257,14 @@
     <string name="biometric_prompt_private_key_title">Authenticate to view private key</string>
     <string name="biometric_auth_error">Authentication failure</string>
     <string name="biometric_auth_error_reason">Authentication failure: %s</string>
+    <string name="wifi_settings">Wi-Fi Settings</string>
+    <string name="wifi_auto_disconnect_checkbox">Disconnect when using to one of these networks:</string>
```

Comment:
I suppose the `to` is a remnant from a previous iteration of this label (`..when connected to...`) and should be removed now.